### ISSUE TITLE
Simulate projector with filter

### ISSUE THREAD

**Issue by cassidyjames** [Philip-Scott/Spice-up#290]
This probably requires GTK 4 and like, GPU filters or something, but it was an idea I had from seeing some slides at LAS. 😉

Projectors almost universally have a poor contrast ratio/are washed out and don't reproduce color as well. I think a desaturate and lower contrast filter when designing slides could be super handy for speakers to see if their design is going to work well on most projectors.

**Comment by Philip-Scott** [comment]
I wonder "how much contrast" would you need in order for it to work with most projectors. I think it's possible to do with some X-Server tinkering, but I also wonder if we could integrate with apps like Harvey to allow you to see if your text is accessible or not (assuming accessible text = can be viewed in projectors)

**Comment by cassidyjames** [comment]
Yeah implementing contrast ratios inside of Spice-up could also work, but I think it would probably hit home more if there was an explicit "simulate projector" button that could be used to see images, backgrounds, etc. all as if they were washed out on a projector.
